### PR TITLE
Roadmap-page-updates-sprint-12

### PR DESCRIPTION
**Pull request by bonnieAcameron**
Updates the Roadmap page according to Elisa & Bonnie's draft.

[Preview link for Roadmap](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/roadmap-page-updates-sprint-12/about/product-roadmap/)

### CODE REVIEW

File: pages/whats-new/product-roadmap.md [hunk 1]
```diff
@@ -11,96 +11,89 @@ redirect_from:
   - /whats-new/product-roadmap/
 ---
 
-Below is our product roadmap: a long-term plan of the goals, features, and long-term direction of USWDS. We update this frequently with the status of our progress, as well as add new high-level feature requests and ideas.
+Here, you’ll find our product roadmap — an up-to-date report on the work we’re doing.
+
+This roadmap communicates what we've recently done and what we've got planned in the next 4-8 months. We update this page every 4 months (our roadmap planning increments). 
+
+Check out our <a href="https://github.com/orgs/uswds/projects/8/views/1">Public GitHub Roadmap</a>for more details about our work.
```
Comment (mejiaj) [inline pages/whats-new/product-roadmap.md:18]: Just confirming. Is the leaderboard tab of the new project board the correct link?

Comment (bonnieAcameron) [reply pages/whats-new/product-roadmap.md:18]: I believe so, but @elisaachen can confirm

Comment (elisaachen) [reply pages/whats-new/product-roadmap.md:18]: > Just confirming. Is the leaderboard tab of the new project board the correct link?

It should be pointing to our new public roadmap board here https://github.com/orgs/uswds/projects/13/views/1

**Comment by thisisdano** [comment]
I removed the manual roadmap from this page and included only links to relevant github boards and views

File: pages/whats-new/product-roadmap.md (hunk 1 above)
Comment (thisisdano) [reply pages/whats-new/product-roadmap.md]: I've updated to the correct link

**Review by thisisdano — APPROVED** [review]

**Pull request merged**